package.json

```diff
@@ -13,28 +16,25 @@
     "lint": "prettier --plugin-search-dir . --check . && eslint \".\"",
     "prepare": "husky install",
     "repo-clean": "git clean -xdf",
-    "test": "jest"
+    "test": "pnpm run build && vitest run"
```

Comment:
actually there is no need to run build before running vitest, in case you want to have it to make sure that the whole project builds before testing it could also make sense

Reply:
Yeah, I like building before testing, but not necessary.   Going to keep it for now.